### DISCUSSION TITLE
Addon does not work, help)

### DISCUSSION THREAD

**Discussion by Eweq322** [ross-g/io_pdx_mesh#78] [Q&A] [answered]
Hello. Decided to do mods on Hoi4. Got to the model editor, wanted to make their own models, began to understand everything). 
Came across your addon to Blender, did everything according to the instructions, but nothing worked, it is installed, like even runs (I looked in the console). I can't import or export Hoi4 model files, there is a checkmark on the addon, shows it's on, but nothing, and no toolbar either. I've tried both new and old versions of Blender, but the result is
![Безымянный123](https://user-images.githubusercontent.com/125033970/230642978-6a52c02d-936c-4eae-ac67-91d118e16ea2.png)
 the same. Can you tell me what my problem might be?

**Comment by ross-g** [comment, marked as answer]
If you open the `Sidebar` in a 3D viewport
![image](https://user-images.githubusercontent.com/9258768/232104975-c686680f-271a-43b2-a63b-f8f0c047c272.png)

You should get a panel for the addon
![image](https://user-images.githubusercontent.com/9258768/232105171-a4397899-02d1-40f0-b264-399e02b6b7e8.png)

Does that work for you?

**Reply by Eweq322** [reply]
Oh yes, everything is in place, everything works, thank you very much. You're a cool man)
